### PR TITLE
Add reference to ping options

### PR DESCRIPTION
**Pull request by jpbede**
## Proposed change
Add a reference to the ping options, the existing ping count and the newly added consider home interval (https://github.com/home-assistant/core/pull/104881).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104881
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ping.markdown [hunk 1]
```diff
@@ -25,6 +25,14 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+## Integration options
+
+It is possible to change some behaviors through the integration options.
+To change the settings, go to {% my integrations title="**Settings** > **Devices & Services**" %}. Select the **Ping** integration, then select **Configure**.
+
+- **Ping count**: Number of echo requests to send to the target. The default is 5.
+- **Consider home**: Number of seconds that must elapse before considering a disconnected device "not at home". The default is 180 seconds (3 minutes).
```
Comment (c0ffeeca7) [inline source/_integrations/ping.markdown:45]:
```suggestion
1. To change the settings, go to {% my integrations title="**Settings** > **Devices & Services**" %}. 
2. Select the **Ping** integration, then select **Configure**.
    - **Ping count**: Number of echo requests to send to the target. The default is 5.
    - **Consider home**: Number of seconds that must elapse before considering a disconnected device "not at home". The default is 180 seconds (3 minutes).
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @jpbede :+1:

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @jpbede :+1:

**Comment by c0ffeeca7** [comment]
@jpbede, there is a merge-conflict.

**Comment by jpbede** [comment]
Oh yes, forgotten to rebase after last ping change, thanks for hinting @c0ffeeca7. Fixed.

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**